pkg/controller/pxc/controller.go

```diff
@@ -218,6 +218,13 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(request reconcile.Request) (re
 		}
 	}()
 
+	if o.CompareVersionWith("1.7.0") >= 0 {
```

Comment:
according to product requirements we should prerform recovery only if `spec.pxc.autoRecovery == true`
```suggestion
	if o.CompareVersionWith("1.7.0") >= 0 && o.pxc.autoRecovery {
```

Reply:
ed447bbb